src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+    border: 1px solid $white-gray-color;
+    width: 166px;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: space-between;
+
+    &__image {
+      height: 134px;
+      margin-bottom: 40px;
+      align-self: center;
+    }
+
+    &__name {
+      width: 166px;
+      margin-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $primary-color;
+    }
+
+    &__code {
+      width: 98px;
+      margin-bottom: 16px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $secondary-color;
+    }
+
+    @import 'stars';
```

Comment:
We should put our imports on top of the file, it makes code more reliable. 

Let's move this import to index.scss